src/notedata.h

```diff
@@ -4,6 +4,15 @@
 #include <QObject>
 #include <QDateTime>
 
+struct NoteExport
+{
+    QString id;
+    QString fullTitle;
+    QDateTime lastModificationDateTime;
+    QDateTime creationDateTime;
+    QString content;
+};
+
```

Comment:
@theshadowx, I used the struct to ensure that we serialize only the data necessary to reconstruct the note. NoteData is a QObject which has additional properties not related to the note itself.